sdks/python/apache_beam/metrics/cells.py

```diff
@@ -31,7 +31,9 @@
 
 
 class CellCommitState(object):
```

Comment:
It does not look like there is much public interfaces in this file. Should we add `__all__`?